core/server/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -59,7 +59,7 @@
   public static final String GET_VERSION = "version";
   public static final String GET_METRICS = "metrics";
 
-  private final AlluxioWorker mWorker = AlluxioWorker.get();
+  private final AlluxioWorker mWorker = DefaultAlluxioWorker.get();
```

Comment:
I decided not to go this route since we run the main class of `DefaultAlluxioWorker` directly, and it seems weird for it to call a factory method on the `AlluxioWorker` interface to create an instance of itself. The line in question is https://github.com/Alluxio/alluxio/pull/3736/files#diff-9a3538ffacd52c73c1f68a27b556d44dR82